CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## [Unreleased]
 
+## [2.0.0-rc.1.1] - 2025-02-08
+
+### Added
+
+- add `iroha transaction get` and other important commands (#5289)
```

Comment:
I'm not sure if this follows the [backporting workflow](https://github.com/hyperledger-iroha/iroha/issues/4570#issuecomment-2405233995) @mversic 

Also,

> 2. Update `CHANGELOG` in a separate commit (we must have an entry for patch versions in the `main` branch as well)

Why do we update the `CHANGELOG.md` in the main branch for every minor version change due to backporting? Wouldn't it be sufficient to update it only in the release branches?

Reply:
> Why do we update the `CHANGELOG.md` in the main branch for every minor version change due to backporting? Wouldn't it be sufficient to update it only in the release branches?

all the history needs to be tracked in the main branch. Other branches are not the source of truth